src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -35,3 +35,12 @@ function has_remote_url {
     echo true
   fi
 }
+
+# Returns true if the repository has an upstream remote configured
+function has_remote_upstream {
+  if [ "$(git remote -v | grep -c 'upstream')" == 0 ]; then
```

Comment:
What about being a bit more specific here, like using `'^upstream\b` as the regex for grep? Otherwise it would match "whatsupstream" or "upstream_old" here.